types/office-js-preview/index.d.ts

```diff
@@ -20018,7 +20018,11 @@ declare namespace Excel {
     }
     /**
      *
-     * Provides information about the binding that raised the SelectionChanged event.
+     * Provides information about the selection that raised the "SelectionChanged" event.
```

Comment:
If it's the name of the event, then I suggest:
```suggestion
     * Provides information about the selection that raised the `SelectionChanged` event.
```

Reply:
This and other instance.

Reply:
I'm using quotation marks to correlate to the literals of the `EventType` enum.